lib/cws/server.ts

```diff
@@ -98,8 +98,8 @@ export class WebSocketServer extends EventEmitterServer {
           secure: !!(req.connection instanceof TLSSocket && (req.connection.authorized || req.connection.encrypted))
         };
 
-        return configs.verifyClient(info, (result: any, code: number, name: string) =>
-          result ? this.handleUpgrade(req, socket) : this.dropConnection(socket, code, name));
+        return configs.verifyClient(info, (clientVerified: boolean, code: number = 500, name: string = 'Client verification failed') =>
```

Comment:
500 is internal server error i would probably put 401 as a default close code (also will match with ws.js)

Reply:
Ok, I have changed the default code to 401.